kubernetes/nodecollection_test.go

```diff
@@ -71,13 +71,15 @@ func TestEnsureNodeSource(t *testing.T) {
 	// The final 400 fails the direct connection test, and as no proxy client is provided
 	// for the "unsuccessful" node test, it falls through to unreachable status.
 	// If no status code is provided, the default response is 200.
-	returnCodes := []int{200, 200, 400, 400, 200, 200, 400}
+	returnCodes := []int{200, 200, 200, 400, 400, 400, 200, 200, 200, 400}
```

Comment:
The ordering in this list is starting to get unwieldy. We'll probably want to do something a little more readable the next time we add something, but that's a problem for _future us_